perception_control:
  bb_controllers:
    proportional:
      downcam:
        drive_max_setpoint: 1.0
        strafe_max_setpoint: 1.0
        drive_maxout_pixel_dist: 200
        strafe_maxout_pixel_dist: 200